src/net-mdns/Spike/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Common.Logging;
+using Common.Logging.Simple;
+using Makaretu.Dns;
+using System;
+using System.Linq;
+
+namespace Spike
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Multicast DNS spike");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)